index.html

```diff
@@ -0,0 +1,92 @@
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="./reset.css" />
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="/">
+          <img src="./img/Property 1=lg.png" alt="logo" />
+        </a>
+        <button><a href="./login.html">로그인</a></button>
+      </nav>
+    </header>
+    <div class="main-wrapper">
+      <section class="banner">
+        <div id="banner-content">
+          <h1>일상의 모든 물건을<br />거래해 보세요</h1>
+          <button><a href="./items.html">구경하러 가기</a></button>
+        </div>
+        <img src="./img/Img_home_top.png" alt="banner-top" />
+      </section>
+      <main>
+        <section class="card-box">
+          <div class="card">
+            <img src="./img/Img_home_01.png" alt="hot-item-image" />
+            <div>
+              <span class="card-title">Hot item</span>
+              <h2>인기 상품을<br />확인해 보세요</h2>
+              <p>
+                가장 HOT한 중고거래 물품을<br />
+                판다 마켓에서 확인해 보세요
+              </p>
+            </div>
+          </div>
+        </section>
+        <section class="card-box">
+          <div class="card">
+            <div>
+              <span class="card-title">Search</span>
+              <h2>구매를 원하는<br />상품을 검색하세요</h2>
+              <p>
+                구매하고 싶은 물품은 검색해서<br />
+                쉽게 찾아보세요
+              </p>
+            </div>
+            <img src="./img/Img_home_02.png" alt="search-image" />
+          </div>
+        </section>
+        <section class="card-box">
+          <div class="card">
+            <img src="./img/Img_home_03.png" alt="" />
+            <div>
+              <span class="card-title">Register</span>
+              <h2>판매를 원하는<br />상품을 등록하세요</h2>
+              <p>어떤 물건이든 판매하고 싶은 상품을 <br />쉽게 등록하세요</p>
+            </div>
+          </div>
+        </section>
+      </main>
+      <section class="banner">
+        <h1>믿을 수 있는 <br />판다마켓 중고 거래</h1>
+        <img src="./img/Img_home_bottom.png" alt="banner-bottom" />
+      </section>
+      <footer>
+        <span class="copyright">©codeit - 2024</span>
+        <div class="infos">
+          <span class="info"><a href="./privacy.html">Privacy Policy</a></span>
+          <span class="info"><a href="./faq.html">FAQ</a></span>
+        </div>
+        <div>
+          <a href="https://www.facebook.com/?locale=ko_KR" target="_blank">
+            <img class="sns" src="./img/ic_facebook.png" alt="facebook" />
+          </a>
+          <a href="https://x.com/" target="_blank">
+            <img class="sns" src="./img/ic_twitter.png" alt="twitter" />
+          </a>
+          <a href="https://www.youtube.com" target="_blank">
+            <img class="sns" src="./img/ic_youtube.png" alt="youtube" />
+          </a>
+          <a href="https://www.instagram.com/" target="_blank">
+            <img class="sns" src="./img/ic_instagram.png" alt="instagram" />
+          </a>
+        </div>
```

Comment:
💊 제안
이 영역은 로고들의 모음이라고 할 수 잇습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
        <ul>
          <li>
            <a href="https://www.facebook.com/?locale=ko_KR" target="_blank">
              <img class="sns" src="./img/ic_facebook.png" alt="facebook" />
            </a>
          </li>
        </ul>
```